tests/test_samples.py

```diff
@@ -196,6 +197,39 @@ def test_plot_ci_par_func(is_par, plot_par, compute_on_par, geometry):
         samples.plot_ci(plot_par=plot_par)
 
 
+@pytest.mark.parametrize("geometry",
+                         [cuqi.geometry.Discrete(4),
+                          cuqi.geometry.KLExpansion(np.linspace(0, 1, 4)),
+                          cuqi.geometry.Continuous2D((2, 2))])
```

Comment:
Add test for Image2D geometry

Reply:
Thank you @nabriis catching this case (Image2D) in the documentation and the test 👍. I added a test case for it.